blog/_posts/2016-09-30-Thank-you-all-from-melbourne.md

```diff
@@ -0,0 +1,50 @@
+---
```

Comment:
Could you make the filename and permalink and title consistent? So either change the filename so it fits the permalink, or vice-versa.